Integrations/Whois/Whois.py

```diff
@@ -8285,7 +8285,7 @@ def whois_command():
                 '%d-%m-%Y')
             md['Expiration Date'] = whois_result.get('expiration_date')[0].strftime('%d-%m-%Y')
     except ValueError as e:
-        return_error('Date could not be parsed. Please check the date again.\n{error}'.format(type(e)))
+        return_error('Date could not be parsed. Please check the date again.\n{}'.format(e))
```

Comment:
i think this might cause issues cause of unicode
better to format as `str(e)`

Reply:
It's the same. `format` internally does the same as `str` to the passed object.